### ISSUE TITLE
"Featured sessions" clue button is not working

### ISSUE THREAD

**Issue by VasBu** [OpenSecuritySummit/oss2018#706]
![image](https://user-images.githubusercontent.com/15072044/40704057-6d248e28-63e7-11e8-9f17-eb7aa0d78814.png)

**Comment by fzipi** [comment]
I don't think it was meant to work, just to inform that those were the "featured" ones. @DinisCruz ?
